### PR TITLE
Update how-to-apply-for-teacher-training.md

### PR DESCRIPTION
**Pull request by emilytraff**
Editing for a better structure and more info.

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
@emilytraff This hasn't built properly - looks to me like this is because the site domain is included in internal links

eg `https://getintoteaching.education.gov.uk/events` rather than `/events`

Do you want to try changing these and seeing if it will rebuild then?

Three other initial thoughts:

**1. Tags**

One of the tags on this blog post is `applying for teacher training` but it is the only blog post with this tag.
Suggest it is changed to `applications` which is used on a few other blog posts.

**2. Date**

Might be worth updating the date so it goes to the top of the blog?

**3. Footer**

There is guidance in https://github.com/DFE-Digital/get-into-teaching-app/blob/master/docs/content.md#creating-a-blog-post on blog footers - it's not quite right at the moment!

image

Let us know when you've updated the post and we'll review/publish 

cc @EllieNodder

**Review by emilytraff — COMMENTED** [review]
Attempting rebuild

**Comment by emilytraff** [comment]
Thanks so much @EllieNodder - is there anything more I need to do?

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 1]
```diff
@@ -24,47 +24,73 @@ tags:
...
+
+Whilst school experience isn’t compulsory, it’s a good idea to visit a school, if only to confirm that you are making the right career choice. Use our [Get School Experience service](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
+
+## 3.	Get a teacher training adviser
+
+Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application. [Get an adviser.](https://adviser-getintoteaching.education.gov.uk/)
+
+## 4.	Attend a teacher training event
+
+Our free events are the perfect opportunity to find out more about teacher training. Chat to teachers, training providers and expert advisers, and get answers to all of your questions. [Find an event.](/events)
```
Comment (EllieNodder) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:60]:
```suggestion
Our free events are the perfect opportunity to find out more about teacher training. Chat to teachers, training providers and expert advisers, and get answers to all of your questions. 

[Find an event.](/events)
```

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 2]
```diff
@@ -24,47 +24,73 @@ tags:
...
+
+If you’re applying for teacher training as an international candidate, [find out more about overseas qualifications.](/train-to-teach-in-england-as-an-international-student).
+
+You can get advice about your qualifications by [chatting to a teacher training adviser.](/teacher-training-advisers)
+
+## 2. Get some school experience
+
+If you haven’t had any experience of working with children, think about any transferable skills you have gained which could be used in a classroom, such as communication, organisation and time management. You might have organised seminars or delivered training sessions, or you might have qualifications that could be used in an educational setting.
+
+Whilst school experience isn’t compulsory, it’s a good idea to visit a school, if only to confirm that you are making the right career choice. Use our [Get School Experience service](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
```
Comment (EllieNodder) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:52]:
```suggestion
Whilst school experience isn’t compulsory, it’s a good idea to visit a school, if only to confirm that you are making the right career choice. You can [search for and request school experience](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
```
Could we rewrite this bit to integrate the service more into the text?

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 3]
```diff
@@ -24,47 +24,73 @@ tags:
...
+## 9.	Complete your application
 
-### 3. Arrange your references
+You’re now ready to submit your application and [apply for teacher training.](https://www.gov.uk/apply-for-teacher-training)
 
-Your referees can tell training providers about your qualities and abilities, so it’s important they know you well. Find out more about the requirements for references in our [tips on applying for teacher training](/tips-on-applying-for-teacher-training).
+Your application will appear in your chosen providers’ inbox as soon as you click submit, but don’t expect an instant response. Wait for them to consider your application and set up an interview. This can take some time to organise.
 
-### 4. Apply for teacher training
+Be patient and do attend all of your interviews, if you have been offered them, before making any final decisions. [Get some top tips for interview success.](/blog/11-top-tips-for-interview-success)
```
Comment (EllieNodder) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:94]:
```suggestion
Be patient and do attend all of your interviews, if you have been offered them, before making any final decisions.

[Get some top tips for interview success.](/blog/11-top-tips-for-interview-success)
```

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 4]
```diff
@@ -24,47 +24,73 @@ tags:
...
 
-### 2. Get one-to-one support from our teaching experts
+## 6.	Find the right training course for you
 
-Our teacher training advisers are all experienced teachers ready to support your journey into teaching. [Register with us](/tta-service) to be allocated your very own adviser.
+Use [Find postgraduate teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk/) to search available courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have three choices for teacher training providers. It’s a good idea to select three choices; the more flexible you are, the more likely it is that you’ll secure a place.
 
-### 3. Attend a virtual teacher training event
+When choosing a course provider for your teacher training, it’s important to consider what will suit your individual circumstances. [Get advice on choosing a provider.](/blog/choosing-the-right-teacher-training-course-provider)
```
Comment (EllieNodder) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:76]:
```suggestion
When choosing a course provider for your teacher training, it’s important to consider what will suit your individual circumstances.

[Get advice on choosing a provider.](/blog/choosing-the-right-teacher-training-course-provider)
```

**Comment by emilytraff** [comment]
Thanks so much @EllieNodder - I've accepted all of those. I think the only one outstanding is about capitalisations on finance but happy to take your lead on that. Please let me know if there's anything more I need to do.

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 5]
```diff
@@ -24,47 +24,73 @@ tags:
...
+
+Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get help with things like finding school experience, choosing the right course for you, funding, and writing a successful application. [Get an adviser.](https://adviser-getintoteaching.education.gov.uk/)
+
+## 4.	Attend a teacher training event
+
+Our free events are the perfect opportunity to find out more about teacher training. Chat to teachers, training providers and expert advisers, and get answers to all of your questions. [Find an event.](/events)
+
+## 5.	Explore your funding options
+
+There are plenty of funding options available to support you during your teacher training year. You can access Tuition Fee and Maintenance Loans regardless of whether or not you have an existing student loan and you will only begin to repay these once you’re earning more than £27,295.
```
[the remark this answers is not in the document]
Comment (EllieNodder) [reply app/views/content/blog/how-to-apply-for-teacher-training.md:68]: Thanks Emily - I think maybe we go with lowercase in this context to match our style elsewhere? :)

**Review by emilytraff — COMMENTED** [review]
Thanks @EllieNodder I've changed those capitals.

File: app/views/content/blog/how-to-apply-for-teacher-training.md [hunk 6]
```diff
@@ -24,47 +24,82 @@ tags:
...
+
+Extra financial support is available if you’re a parent, have an adult dependant or a disability. This support does not need to be paid back.
+
+And for some subjects, you may qualify for a tax-free bursary or scholarship.
+
+The key to managing financially during your training year is careful planning and preparation. Our [expert advisers](https://adviser-getintoteaching.education.gov.uk/) can help you make a plan and find the best funding options for your circumstances. [Explore your funding options.](/funding-your-training)
+
+## 6.	Find the right training course for you
+
+[Find postgraduate teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk/) courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have three choices for teacher training providers. It’s a good idea to select three choices; the more flexible you are, the more likely it is that you’ll secure a place.
```
Comment (EllieNodder) [inline app/views/content/blog/how-to-apply-for-teacher-training.md:78]:
```suggestion
[Find postgraduate teacher training](https://www.find-postgraduate-teacher-training.service.gov.uk/) courses in England. You’ll be able to search by location, training provider and subject. When you apply, you have 3 choices for teacher training providers. It’s a good idea to select 3 choices; the more flexible you are, the more likely it is that you’ll secure a place.
```
Sorry, one last change!! Can we just make these into numerals pls

**Comment by EllieNodder** [comment]
@peteryates not sure why this has failed? Will you be able to take a look when you get chance pls

**Comment by peteryates** [comment]
@EllieNodder - rebased, hopefully it'll work now. Looks like we hit a rate limit on Snyk (our security testing service)

**Comment by emilytraff** [comment]
Hi @peteryates - there still seems to be something stopping this from publishing?

**Comment by peteryates** [comment]
Just a review I think. @EllieNodder - good to go?

**Comment by EllieNodder** [comment]
> Just a review I think. @EllieNodder - good to go?

@emilytraff @peteryates just need @gemmadallmandfe to review and we can publish 😃

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**